pallets/rc-migrator/src/weights_ah.rs

```diff
@@ -0,0 +1,226 @@
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Holds` (r:255 w:255)
+	/// Proof: `Balances::Holds` (`max_values`: None, `max_size`: Some(67), added: 2542, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Freezes` (r:255 w:255)
+	/// Proof: `Balances::Freezes` (`max_values`: None, `max_size`: Some(67), added: 2542, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Locks` (r:255 w:255)
+	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1299), added: 3774, mode: `MaxEncodedLen`)
+	/// The range of component `n` is `[1, 255]`.
+	fn receive_accounts(n: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `15396 + n * (1310 ±0)`
+		//  Estimated: `990 + n * (3774 ±0)`
+		// Minimum execution time: 89_000_000 picoseconds.
+		Weight::from_parts(90_000_000, 990)
+			// Standard Error: 217_672
+			.saturating_add(Weight::from_parts(86_684_960, 0).saturating_mul(n.into()))
+			.saturating_add(T::DbWeight::get().reads((4_u64).saturating_mul(n.into())))
+			.saturating_add(T::DbWeight::get().writes((4_u64).saturating_mul(n.into())))
+			.saturating_add(Weight::from_parts(0, 3774).saturating_mul(n.into()))
+	}
+	/// Storage: `Claims::Vesting` (r:255 w:255)
+	/// Proof: `Claims::Vesting` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// The range of component `n` is `[1, 255]`.
+	fn receive_claims(n: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `43`
+		//  Estimated: `1033 + n * (2475 ±0)`
+		// Minimum execution time: 10_000_000 picoseconds.
+		Weight::from_parts(2_353_862, 1033)
+			// Standard Error: 20_936
+			.saturating_add(Weight::from_parts(2_030_192, 0).saturating_mul(n.into()))
+			.saturating_add(T::DbWeight::get().reads((1_u64).saturating_mul(n.into())))
+			.saturating_add(T::DbWeight::get().writes((1_u64).saturating_mul(n.into())))
+			.saturating_add(Weight::from_parts(0, 2475).saturating_mul(n.into()))
+	}
+	/// Storage: `Proxy::Proxies` (r:0 w:255)
+	/// Proof: `Proxy::Proxies` (`max_values`: None, `max_size`: Some(1241), added: 3716, mode: `MaxEncodedLen`)
+	/// The range of component `n` is `[1, 255]`.
+	fn receive_proxy_proxies(n: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 9_000_000 picoseconds.
+		Weight::from_parts(10_000_000, 0)
+			// Standard Error: 21_936
+			.saturating_add(Weight::from_parts(2_351_242, 0).saturating_mul(n.into()))
+			.saturating_add(T::DbWeight::get().writes((1_u64).saturating_mul(n.into())))
+	}
+	/// Storage: `System::Account` (r:255 w:255)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// The range of component `n` is `[1, 255]`.
+	fn receive_proxy_announcements(n: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `15396 + n * (1310 ±0)`
+		//  Estimated: `990 + n * (2603 ±0)`
```

Comment:
NOTE PoV weights have a higher than expected base, but the linear component is smaller.